### ISSUE TITLE
Incorrect production mode usage example?

### ISSUE THREAD

**Issue by copygirl** [jhbertra/express-ipn#1]
In the readme, you show the following example for turning on production mode:
```
app.post('/', ipn.validator(validationHandler), true);
```
Shouldn't that instead be the following?
```
app.post('/', ipn.validator(validationHandler, true));
```

**Comment by Shropford** [comment]
Yes, that stopped it crashing me - Thanks - Note to self: Check issues quicker...

**Comment by jhbertra** [comment]
Thanks for spotting that, fixed in #4
